lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -359,11 +375,28 @@ instance FromJSON WalletPutData where
 instance ToJSON  WalletPutData where
     toJSON = genericToJSON defaultRecordTypeOptions
 
+instance FromJSON PostExternalTransactionData where
+    parseJSON = withObject "PostExternalTransactionData" $ \o -> do
+        (Aeson.String load) <- o .: "payload"
```

Comment:
I'm pretty sure that this can decode straight to `Text` and so doesn't need the `Aeson.String` constructor.
```suggestion
        load <- o .: "payload"
```

Reply:
yes, you are right. corrected